ui/raidboss/data/triggers/twinning.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
+
+// The Twinning
+
+[{
+  zoneRegex: /^The Twinning$/,
+  triggers: [
+    {
+      // The handling for these mechanics is similar enough it makes sense to combine the trigger
+      id: 'Twinning Impact + Pounce',
+      regex: / 1B:........:\y{Name}:....:....:(003[2-5]|005A)/,
+      suppressSeconds: 10,
+      alertText: {
+        en: 'Spread--Avoid cages',
+      },
+    },
+    {
+      id: 'Twinning Beastly Roar',
+      regex: / 14:3D64:Alpha Zaghnal starts using Beastly Roar/,
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      alertText: {
+        en: 'AoE Damage',
+      },
+    },
+    {
+      id: 'Twinning Augurium',
+      regex: / 14:3D65:Alpha Zaghnal starts using Augurium on (\y{Name})/,
+      alertText: function(data, matches) {
+        if (data.me == matches[1]) {
+          return {
+            en: 'Buster cone on YOU',
+          };
+        }
+        return {
+          en: 'Avoid buster cleave',
+        };
+      },
+    },
+    {
+      id: 'Twinning Charge Eradicated',
+      regex: / 1B:........:(\y{Name}):....:....:005D/,
+      alertText: function(data, matches) {
+        if (data.me == matches[1]) {
+          return {
+            en: 'Stack on YOU',
+          };
+        }
+        return {
+          en: 'Stack on ' + data.ShortName(matches[1]),
+        };
+      },
+    },
+    {
+      id: 'Twinning adds',
+      regex: / Added new combatant/,
```

Comment:
Does this not pop all the time?

Reply:
That's a good point. I clearly didn't think this one through. I don't have a recent log with broken cages, so I may have to grab a group to check the exact log lines. I'll remove that for now.